macros/Compat.ejs

```diff
@@ -406,31 +376,61 @@ function getData(queryString, obj) {
   }, obj);
 }
 
-var data = getData(query, bcd.data);
+/*
+Get features that should be displayed according to the query and the depth setting
+Flatten them into a features array
+*/
+function traverseFeatures(obj, depth, identifier) {
+  depth--;
+  if (depth >= 0) {
+    for (i in obj) {
+      if (!!obj[i] && typeof(obj[i])=="object" && i !== '__compat') {
+        if (obj[i].__compat) {
+          // [identifier + '.' + i]
```

Comment:
I don't understand this comment, and I'm not sure what `identifier` is doing here in general. Should `[i]` in the next line be `[identifier]`, so you get dotted identifiers for child features?